cmd/tests/cmd_run_test.go

```diff
@@ -2027,6 +2027,21 @@ func TestEventSystemError(t *testing.T) {
 			},
 			expExitCode: exitcodes.ScriptException,
 		},
+		{
+			name: "invalid",
+			script: `
+				export const options = {
+					vus: 'this is an invalid type',
+				}
+				
+				export default function () {}
+			`,
+			expLog: []string{
+				`got event Exit with data '&{Error:could not initialize '-': could not load JS test 'file:///-': strconv.ParseInt: parsing \"this is an invalid type\": invalid syntax}'`,
```

Comment:
I tried a multiline string here, but couldn't correctly assert it. If there's a correct way of doing this let me know!